.yarnrc.yml

```diff
@@ -4,4 +4,4 @@ enableGlobalCache: false
 
 nodeLinker: node-modules
 
-yarnPath: .yarn/releases/yarn-4.3.0.cjs
+yarnPath: .yarn/releases/yarn-4.2.2.cjs
```

Comment:
we got many peer deps warning due to [rework](https://github.com/yarnpkg/berry/pull/6205) shipped in 4.3.0